README.md

```diff
@@ -2001,6 +2001,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Tomo Bones](https://github.com/tomobones)
   - [Triman Kaur](https://github.com/Trimankaur)
   - [Tsipporah Christopher](https://github.com/tsipporahc)
+  - [Tshepiso Lehutjo](https://github.com/tshepi-tech)
```

Comment:
AFAIK, your name must be in line 2003, after Triman Kaur (`H` comes first, then `I`).
Let us know if you need any help with that. 🙂